tox.ini

```diff
@@ -46,17 +46,9 @@ basepython = python2.7
 commands =
     sphinx-build -W -b linkcheck docs docs/_build/html
 
-# Temporarily disable coverage on pypy because of performance problems with
+# This target disables coverage on pypy because of performance problems with
 # coverage.py on pypy.
-[testenv:pypy]
-commands =
-    pip list
-    python -c "from cryptography.hazmat.backends.openssl.backend import backend; print(backend.openssl_version_text())"
-    py.test --capture=no --strict {posargs}
-
-# Temporarily disable coverage on pypy3 because of performance problems with
-# coverage.py on pypy3.
-[testenv:pypy3]
+[testenv:pypy-nocoverage]
```

Comment:
This needs `basepython = pypy`, otherwise it's just running CPython I assume :-)